src/bridge/components/StatelessTransaction.js

```diff
@@ -358,7 +358,7 @@ class StatelessTransaction extends React.Component {
         onClick={ setUserApproval }
         state={ userApproval }
       >
-        <div>
+        <div onClick={ setUserApproval }>
```

Comment:
Sorry to keep hitting you with the refactoring comments, but:

Shouldn't checkbox labels just be part of the `CheckboxButton` component (and implemented as `<label>` for native click-ability)?

Reply:
Haha, that's okay. 

The problem is that native-label-clickability works when it's attached to an `<input>` with a 'for' attribute. CheckboxButton is implemented as a `<button>` with a custom way of displaying the 'checked' state (TFW no native component styling). Refactoring CheckboxButton seems like a lot of work for the functionality we want to accomplish here.

Reply:
I just looked at the code, as I should have earlier, and... oh my, raw SVG, how lovely!

Fair re: native `label`ing. I still think there's something to be said for moving labels into the components, but I'm not dead-set of having that done right now.

It's weird though that we have both a `Checkbox` and a `CheckboxButton`, and that they're completely distinct. The `Checkbox` does have built-in labels it seems like? `subtitle` or something.  
Wake me from this nightmare.